rust/basic_ethereum/README.md

```diff
@@ -0,0 +1,179 @@
+---
+keywords: [ advanced, rust, ethereum, eth, integration, ethereum integration ]
+---
+
+# Basic Ethereum
+
+[View this sample's code on GitHub](https://github.com/dfinity/examples/tree/master/rust/basic_ethereum)
+
+## Overview
+
+This tutorial will walk you through how to deploy a
+sample [canister smart contract](https://internetcomputer.org/docs/current/developer-docs/multi-chain/ethereum/overview)
+**that can send and receive Ether (ETH)** on the Internet Computer.
+
+## Architecture
+
+This example internally leverages
+the [threshold ECDSA](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa)
+and [HTTPs outcalls](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview)
+features of the Internet Computer.
+
+For a deeper understanding of the ICP < > ETH integration, see
+the [Ethereum integration overview](https://internetcomputer.org/docs/current/developer-docs/multi-chain/ethereum/overview).
+
+## Prerequisites
+
+* [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
+
+## Step 1: Building and deploying sample code
+
+### Clone the smart contract
+
+To clone and build the smart contract in **Rust**:
+
+```bash
+git clone https://github.com/dfinity/examples
+cd examples/rust/basic_ethereum
+git submodule update --init --recursive
+```
+
+**If you are using MacOS, you'll need to install Homebrew and run `brew install llvm` to be able to compile the example.
+**
+
+### Acquire cycles to deploy
+
+Deploying to the Internet Computer
+requires [cycles](https://internetcomputer.org/docs/current/developer-docs/setup/cycles) (the equivalent of "gas" in
+other blockchains). You can get free cycles from
+the [cycles faucet](https://internetcomputer.org/docs/current/developer-docs/setup/cycles/cycles-faucet.md).
+
+### Deploy the smart contract to the Internet Computer
+
+```bash
+dfx deploy --network ic basic_ethereum --argument '(opt record {ethereum_network = opt variant {Sepolia}; ecdsa_key_name = opt variant {TestKey1}})'
```

Comment:
Note that `--ic` is how the cool kids write `--network ic` nowadays.

Reply:
Thanks for heads-up, done.